super_dpf.py

```diff
@@ -89,8 +92,26 @@ def verify_local(self, cloud_filelist, local_filelist):
                 log.info('Deleting local file {}'.format(local_file))
                 os.remove(local_file)
 
-    def stylize_image(self, image):
-        return image
+    def stylize_image(self, img):
```

Comment:
Has this function been tested on both 4:3 images with a 16:9 output and 16:9 images targeting a 4:3 output?  I'm not sure it will go both directions unless I am missing something.